# This message contains GPS data fields:
std_msgs/Header header

float32 gps_time
geometry_msgs/Vector3 velocity_ned